### DISCUSSION TITLE
How to add icon to button and link

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#3247] [Help] [answered]
Hi guys, i try to find out any way to add icon beside the navigation link and button but no success. Any want have any idea how to  do that?

Tq .

**Comment by mjashanks** [comment, marked as answer]
This is not supported out of the box, but you can build the nav bar up yourself.

1. Go to "Layouts" and click on the relevant layout
2. Set "Navigation" to "None"
3. Add a container and manually fill it with links and icons. My component tree looks like this:

![image](https://user-images.githubusercontent.com/3524181/140297719-e4d1776b-ec29-42ac-af54-7c96ef8cc570.png)

As for adding an icon to a button, it's not possible. Sometimes I use Unicode emojis for the Button text and set the Button to "Quiet", so there is no border.

**Comment by cverdes** [comment]
I think that a simple and fast way to include icons in buttons, could be to include [fontawesome](https://fontawesome.com/v6.0/icons?s=solid%2Cbrands) fonts available in budibase to be able to use the unicodes of the symbols as text.

**Comment by khalid-al-walid** [comment]
1. go to https://apps.timwhitlock.info/emoji/tables/unicode
2. copy any native icon
3. paste anywhere
